frontend/components/Auction.vue

```diff
@@ -203,15 +203,20 @@
             <TextBlock>
                 <div class="flex w-full justify-end flex-wrap mt-4">
                     <Tooltip placement="top">
-                        <div slot="title">
+                        <!--<div slot="title">
                             This website does not yet support bidding on the auction with your own DAI. In the
                             meantime, you can use
                             <a href="https://liquidations.makerdao.com/" target="_blank" class="underline text-primary"
                                 >Liquidation Portal</a
                             >
-                        </div>
+                        </div>-->
                         <div>
-                            <Button disabled type="secondary" class="w-60 mb-4" @click="$emit('purchase')">
+                            <Button
+                                :disabled="!!errorText"
```

Comment:
Currently there is an errorText with `Swap transaction is not possible...` which suppose to only block swap transaction button and not this one.

To solve it, you can have 2 or 3 different computed properties: `auctionError`, `swapTransactionError` and (if exists) `bidTransactionError`. `swapTransactionError` can first check `if(this.auctionError)` (: